kingpin/actors/group.py

```diff
@@ -62,6 +62,14 @@ def __init__(self, *args, **kwargs):
         # Pre-initialize all of our actions!
         self._actions = self._build_actions()
 
+    def get_orgchart(self, parent=''):
+        ret = super(BaseGroupActor, self).get_orgchart(parent=parent)
```

Comment:
This is unique to "Org Chart" behavior. The data supplied to Org Chart is not nested -- it's a single array of every single node. So this particular line defines the data for the group actor itself, such that all the child actors have a parent to point to. So not root -- any group actor.